canonical-web-and-design/ubuntu.com
# Fix typo in /uasd

## Done

- In definitions, turned 'Node' to 'Nodes'

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/uasd
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
